### ISSUE TITLE
cordova-sqlite-storage

### ISSUE THREAD

**Issue by pyummm** [ionic-team/capacitor#244]
npx cap update
```
Found 18 additional Capacitor plugin(s):
     com-sarriaroman-photoviewer
     cordova-pdf-generator
     cordova-plugin-camera
     cordova-plugin-device
     cordova-plugin-document-viewer
     cordova-plugin-email-composer
     cordova-plugin-file
     cordova-plugin-file-opener2
     cordova-plugin-file-transfer
     cordova-plugin-geolocation
     cordova-plugin-ionic
     cordova-plugin-network-information
     cordova-plugin-splashscreen
     cordova-plugin-statusbar
     cordova-sqlite-storage
     info.protonet.imageresizer
     ionic-plugin-keyboard
     onesignal-cordova-plugin

✖ update ios: ENOENT: no such file or directory, stat '/Volumes/hard/IOS/myApp/node_modules/cordova-sqlite-storage/node_modules/cordova-sqlite-storage-dependencies/sqlite3.c'
(node:33902) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ENOENT: no such file or directory, stat '/Volumes/hard/IOS/myApp/node_modules/cordova-sqlite-storage/node_modules/cordova-sqlite-storage-dependencies/sqlite3.c'
(node:33902) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
⠹ update androidFound 0 Capacitor plugin(s):

✔ update android in 406.80ms
```

**Comment by jcesarmobile** [comment]
Which version of the plugin are you using?

**Comment by pyummm** [comment]
i installed all fresh, as per instructions, just plugins list as i use in live proj.

"cordova-sqlite-storage@2.2.1"

**Comment by IT-MikeS** [comment]
Fixed as of #257 ?

**Comment by jcesarmobile** [comment]
That just removed the error, https://github.com/ionic-team/capacitor/pull/264 makes it work on Android.
It's not working fine on iOS, open the DB successfully, but then it's not able to write on it because it says it's not open. Will look into that, but this can be closed.
